i18n/en.json

```diff
@@ -154,6 +154,7 @@
   "I18N_PLAYER_COMMUNITY_EDITABLE_TOOLTIP": "Community editable",
   "I18N_PLAYER_CONTINUE_BUTTON": "Continue",
   "I18N_PLAYER_CONTINUE_NEXT_CARD": "Continue to next card",
+  "I18N_PLAYER_SHARE_EXPLORATION" : "Enjoyed this exploration? Share it with your friends!",
```

Comment:
@anafrancisca @anggorodewanto Do you have any suggestions for the Portuguese and Indonesian translations? Thanks!